deluge/ui/console/modes/connectionmanager.py

```diff
@@ -93,7 +98,10 @@ def _on_connect_fail(self, result):
             log.exception(result)
 
     def _host_selected(self, selected_host, *args, **kwargs):
-        if selected_host in self.statuses:
+        if (
+            selected_host in self.statuses
+            and self.statuses[selected_host][1] == 'Online'
```

Comment:
I actually think it is useful to retain selecting offline hosts e.g. DNS hostname issue would be displayed...

![image](https://user-images.githubusercontent.com/606038/80388238-8f174300-88a1-11ea-8785-cbd6ba7305ec.png)